### PR TITLE
[Excel] (Custom Functions) Addressing feedback from Customer Issue

### PR DESCRIPTION
**Pull request by keyur32**
Issue: https://github.com/OfficeDev/Excel-Custom-Functions/issues/112

Proposal: Update not to put runtime logging info inline of tutorial

### CODE REVIEW

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3d05d1f](https://github.com/OfficeDev/office-js-docs-pr/pull/746/commits/3d05d1f96c66ecd999a246247ffee39dcac2cea3)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3d05d1f](https://github.com/OfficeDev/office-js-docs-pr/pull/746/commits/3d05d1f96c66ecd999a246247ffee39dcac2cea3)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[3d05d1f](https://github.com/OfficeDev/office-js-docs-pr/pull/746/commits/3d05d1f96c66ecd999a246247ffee39dcac2cea3)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902121655282720-746%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Rp4cOkMLDClH0%2Bv%2BPc8yKikxUKdVU52lp4BJW3Y4l8M%3D&st=2019-02-12T16%3A51%3A45Z&se=2019-03-15T16%3A56%3A45Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mscharlock** [comment]
Just curious - why the change?

File: docs/tutorials/excel-tutorial-create-custom-functions.md [hunk 1]
```diff
@@ -85,7 +85,7 @@ npm run start
 This command starts the web server, and sideloads your custom function add-in into Excel for Windows.
 
 > [!NOTE]
-> If your add-in does not load, check that you have completed step 3 properly.
+> If your add-in does not load, check that you have completed step 3 properly. You can also enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as several installation and runtime conditions. Runtime logging writes `console.log` statements to a log file to help you uncover issues, as well.
```
Comment (davidchesnut) [inline docs/tutorials/excel-tutorial-create-custom-functions.md]:
```suggestion
> If your add-in does not load, check that you have completed step 3 properly. You can also enable **[runtime logging](../testing/troubleshoot-manifest.md#use-runtime-logging-to-debug-your-add-in)** to troubleshoot issues with your add-in's XML manifest file, as well as any installation or runtime problems. Runtime logging writes `console.log` statements to a log file to help you find and fix issues.
```

Comment (davidchesnut) [reply docs/tutorials/excel-tutorial-create-custom-functions.md]: Some suggested rewrites. Hope this works.

Comment (keyur32) [reply docs/tutorials/excel-tutorial-create-custom-functions.md]: cool, I will commit those. @davidchesnut  - can you also make the rewrites to the main trouble shooting guide, I had copied the blurb from there.

**Comment by o365devx** [comment]
OPS Build status updates of commit _[aada56e](https://github.com/OfficeDev/office-js-docs-pr/pull/746/commits/aada56e140311b05f6127d9d266a739ef6b3f424)_: 

:clock10: **Preparing**: average preparing time is _44 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[aada56e](https://github.com/OfficeDev/office-js-docs-pr/pull/746/commits/aada56e140311b05f6127d9d266a739ef6b3f424)_: 

:clock10: **Incremental building**: average incremental building time is _29 sec(s)_

**Comment by o365devx** [comment]
OPS Build status updates of commit _[aada56e](https://github.com/OfficeDev/office-js-docs-pr/pull/746/commits/aada56e140311b05f6127d9d266a739ef6b3f424)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/tutorials/excel-tutorial-create-custom-functions.md](https://github.com/OfficeDev/office-js-docs-pr/blob/keyur32-patch-2/docs/tutorials/excel-tutorial-create-custom-functions.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C2%5C12%5C57856ef7-e8cf-54ca-0d12-df5f42d1e8bb%5CPullRequest%5C201902122253462155-746%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=c4FybZK%2BV%2Boo987wwal31Qu1usq%2F8pCQM5Rga%2F3qs%2FA%3D&st=2019-02-12T22%3A50%3A03Z&se=2019-03-15T22%3A55%3A03Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**